src/microbe_stage/Membrane.cs

```diff
@@ -25,9 +19,8 @@ public class Membrane : MeshInstance, IComputedMembraneData
     /// </summary>
     private readonly List<Vector2> vertices2D = new();
 
-    // Work buffers used when generating membrane data
-    private List<Vector2> previousWorkBuffer = new();
-    private List<Vector2> nextWorkBuffer = new();
+    // Buffer for starting points when generating membrane data
```

Comment:
This should be an xml comment as this is documenting a field

Reply:
Ah, I see. Fixed.